glimr/utils.py

```diff
@@ -133,24 +134,26 @@ def get_trial_info(exp_dir, metric=None):
             result_path = os.path.join(exp_dir, subdir, "result.json")
             if os.path.exists(result_path):
                 df = pd.read_json(result_path, lines=True)
-                df.insert(0, 'trial_#', counter)
+                df.insert(0, "trial_#", counter)
                 dataframes.append(df)
                 counter += 1
-    queried = pd.concat(dataframes, ignore_index=True) 
+    queried = pd.concat(dataframes, ignore_index=True)
     columns = queried.columns
     if metric is not None:
         bool_list = [True if i in columns else False for i in metric]
         if not all(bool_list):
             raise ValueError(f"{metric[bool_list.index(False)]} is not a valid metric")
-        queried = queried.loc[:,metric]
+        queried = queried.loc[:, metric]
     return queried
 
-def prune_constants(space):
-    """Prepares a search space for ray tune's PBT scheduler by pruning constants.
+
+def prune_constants_functions(space):
```

Comment:
Let's change the name here. "prune_search"?

Reply:
Sure!